portfolio/src/main/webapp/index.html

```diff
@@ -1,18 +1,60 @@
 <!DOCTYPE html>
 <html>
   <head>
+    <style>
+body {
+  background-image: url('images/white-cat-1.png');
+}
+</style>
     <meta charset="UTF-8">
     <title>My Portfolio</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
+      <h1>The Portfolio</h1>
+      <b>(Scroll to the bottom for a surprise)</b>
+      <p>Hello, I am Anthony Achille, Check out my <a href="https://github.com/Anthony869?tab=repositories">GitHub</a></p>
+      <p></p>
+      <p></p>
+      <button onclick="addRandomGreeting()">Ricky Quotes</button>
       <div id="greeting-container"></div>
+      <p></p>
+      <p></p>
+      <strong>Background:</strong>  
+      <p>I have always been interested in programming and computers in general. Ever since I could remember I wnated to learn
+         more about computer and how they work. Technology has always been interesting and being able to control them with code.
+         As soon as I could take a class about computers in highschool I did, I took Computer Science 3 the first year and continued 
```

Comment:
"Computer Science 3" - Opportunity to link to the class if your teacher has a webpage for it.  Up to you.